CDS API overview and guidelines pages

afzalmufti opened the pull request:
New info pages for CDS API overview and developer guidelines

Staging links:
https://uec-tech-standards.netlify.com/cds-api-overview.html
https://uec-tech-standards.netlify.com/cds-api-developer-guidelines.html

afzalmufti committed:
Gemfile ruby version

afzalmufti committed:
rollback gemfile edit

afzalmufti committed:
USO-39 Add cds-api demo page

afzalmufti committed:
Update to demo implementation page

afzalmufti committed:
USO-39 Revision to CDS-API demo page

afzalmufti committed:
USO-39 text amend

afzalmufti committed:
Starting new pages for developer self service

afzalmufti committed:
Menu and content changes for cds api

afzalmufti committed:
text amend

afzalmufti committed:
Rework overview wording following team review

afzalmufti committed:
Rework of cds-api overview page, guideline page and glossary

afzalmufti committed:
Diagram update and hiding of some elements

afzalmufti committed:
wording updates

afzalmufti committed:
text amends

afzalmufti committed:
Iteration of elements section in guidelines

afzalmufti committed:
gist code

afzalmufti committed:
swap out inline code for gist hosted

afzalmufti committed:
tidy up sidebar yml

afzalmufti committed:
Merge branch 'publish-upstream'

# Conflicts:
#	_data/sidebars/overview_sidebar.yml

afzalmufti committed:
Merge branch 'publish-upstream'

# Conflicts:
#	_data/sidebars/overview_sidebar.yml
#	images/cds-api-uec-system-architecture.png

afzalmufti committed:
text amend

afzalmufti committed:
`Resource` highlighting

mattstibbs commented on _pages/business_capabilities/cds-api-overview.md:
```diff
@@ -0,0 +1,217 @@
+
+NHS Digital is committed to setting strong, open standards for inter-operable internet-first digital service provision. Development of standards, frameworks and Application Programming Interfaces (APIs) are the chosen route for improving data sharing and giving clinicians better access to patient information.
+
+The [Clinical Decision Support API Specification v1.0.0](https://developer.nhs.uk/apis/cds-api-1-0-0/) is the first standard published to support the interactions between a Clinical Decision Support System (CDSS) and an Encounter Management System (EMS) to support patient triage using open standards.
```
I would suggest a paragraph on "Who might find this useful?" would be a good thing to have

afzalmufti replied:
Summary at top of page: This page is intended as an orientation guide for API developers, who are making systems compliant to the Clinical Decision Support API (CDS API)

mattstibbs commented on _data/glossary.yml:
```diff
@@ -1,9 +1,79 @@
+  expansion: Accident and Emergency
+  definition: An A&E department (also known as emergency department or casualty) deals with genuine life-threatening emergencies.
+- keyword: Alpha
+  definition: A development stage in iterative delivery to build a basic working system with limited functionality used to demonstrate to users 
```
This definition needs improving a little - suggest borrowing from the GDS definition https://www.gov.uk/service-manual/agile-delivery/how-the-alpha-phase-works

afzalmufti replied:
Amended: 
A development stage in iterative delivery where a project team takes new insights and findings from the [discovery](#discovery) phase and begins to experiment with solutions to a challenge. The alpha phase is about rapidly testing ideas to see what works and what doesn't.

Also added entries for discovery and beta:

Discovery
The first stage of development in iterative delivery where a project team analyses a problem and conducts research to understand the users, needs and journeys that need to be considered when developing a solution.

Beta
A development stage in iterative delivery where the learning from the [alpha phase](#alpha) is developed into a working service and tested with real users while minimising risk and maximising the potential to learn and iterate the service.

mattstibbs commented on _data/glossary.yml:
Suggest adding a mention that it is the mandated official clinical terminology for the NHS.

afzalmufti replied:
Amended:
A structured clinical vocabulary for use in an electronic health record. It is the most comprehensive and precise clinical health terminology product in the world. SNOMED CT is to be used as the single terminology in all care settings in England.

mattstibbs commented on _data/glossary.yml:
You should reference that WICs will eventually be replaced by UTCs.

afzalmufti replied:
Amended slightly:
NHS walk-in centres offer convenient access to a range of treatments. WICs are managed by Clinical Commissioning Groups (CCGs), dealing with minor illnesses and injuries. Walk-in centres are gradually being replaced by urgent treatment centres (UTCs).

Beefed up UTC description:

A medical treatment facility where patients can be treated for minor emergencies and injury. UTCs work alongside other parts of the urgent care network including primary care, community pharmacists, ambulance and other community-based services to provide a locally accessible and convenient alternative to A&E for patients who do not need to attend hospital. Commonality of service provision under the UTC name will increasingly end the current confusing range of options and simplify the system so patients know where to go and have clarity of which services are on offer wherever they are in the country.

mattstibbs commented on _pages/emerging/clinical_decision_support_api.md:
```diff
@@ -12,4 +12,6 @@ The guide covers triage of patients by non-clinical staff and triage of patients
 
-[Access the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api/)
\ No newline at end of file
+[Access version 1.0.0 the Clinical Decision Support API implementation guide](https://developer.nhs.uk/apis/cds-api-1-0-0/)
```
**"of"** the Clinical Decision Support API....

afzalmufti replied:
fixed

mattstibbs reviewed (changes requested):
WIthin the CDSS API FAQ section, section "FHIR Primers" - you should link out to the offical documentation on those concepts, not just attempt to define them ourselves.

afzalmufti committed:
Addressing review comments

afzalmufti commented:
Pushed fresh commit addressing all comments with exception of Who might find this useful? paragraph. Hoping summary at the top of the page covers this and is enough until next iteration.
